app/static/horoscope.js

```diff
@@ -0,0 +1,26 @@
+async function daily_horoscope(singName) {
```

Comment:
Prefer to name this function better. Also, it doesn't have to be an async function